### ISSUE TITLE
can't type newlines when `nextEmbedded` is set

### ISSUE THREAD

**Issue by Yokozuna59** [TypeFox/monaco-components#49]
Hi there, in my Monarch syntax highlighter, I have a rule where it uses embedded language:

```typescript
initial: [
  [
    /^\s*%%(?={)/,
    { token: 'keyword', bracket: '@open', next: '@directive', nextEmbedded: 'javascript' },
  ],
  // other rules
]
```

When I enter that state, I can type anything, except newlines. And when I removed `javascript` and made it empty, it worked fine but without the language features.
I tried the code in https://microsoft.github.io/monaco-editor/monarch.html to check if the issue was in my code, but it's working fine.

Here's the minimum Monarch code:

```typescript
export default {
  tokenizer: {
    initial: [
      [
        /^\s*%%(?={)/,
        { token: 'keyword', bracket: '@open', next: '@directive', nextEmbedded: 'javascript' },
      ],
    ],
    directive: [
      [/%%$/, { token: 'keyword', bracket: '@close', next: '@pop', nextEmbedded: '@pop' }],
    ],
  },
};
```

**Comment by kaisalmen** [comment]
Hi @Yokozuna59 could you please share your initialization code (`UserConfig`)? What editor type do you use?

**Comment by Yokozuna59** [comment]
Hi @kaisalmen, I don't have one; it was the generated editor by `generator-langium@2.0.0`.
I'm not sure if it was resolved with the latest version. I tried updating the version, but errors were thrown everywhere, so I stuck with the version that came with `generator-langium`.
I saw that you plan to update the browser template at https://github.com/eclipse-langium/langium/issues/1163, but I'm not sure when it will happen, so I continued working with the version I have.
